src/lib.rs

```diff
@@ -2696,7 +2696,7 @@ pub unsafe extern "C" fn wgpuInstanceRequestAdapter(
                         callback(
                             native::WGPURequestAdapterStatus_Error,
                             std::ptr::null_mut(),
-                            "unsupported backend type: d3d11".as_ptr() as _,
+                            "unsupported backend type: d3d11\0".as_ptr() as _,
```

Comment:
Nit: this uses a c literal which has a \0 automatically added.

```suggestion
                            c"unsupported backend type: d3d11".as_ptr() as _,
```

Reply:
I didn't knew this was a thing, awesome :)